castervoice/apps/explorer.py

```diff
@@ -1,15 +1,42 @@
+# import pyperclip
 from dragonfly import (Grammar, MappingRule, Dictation, IntegerRef,
-                       Repeat, Pause)
+                       Repeat, Pause, Function, Choice)
 
-from castervoice.lib import control
+from castervoice.lib import control, context, utilities, settings
+from castervoice.lib.context import AppContext, paste_string_without_altering_clipboard
 from castervoice.lib import settings
 from castervoice.lib.dfplus.additions import IntegerRefST
 from castervoice.lib.dfplus.merge import gfilter
 from castervoice.lib.dfplus.merge.mergerule import MergeRule
 from castervoice.lib.dfplus.state.short import R
-from castervoice.lib.context import AppContext
 from castervoice.lib.actions import (Key, Text)
 
+# note that the tab structure of Windows Explorer main window is slightly different than 
+# that of Windows Explorer dialogbox (aka child window)
+# this file is only for Windows Explorer main window.
+
+
+# bring me dependencies
+CONFIG = utilities.load_toml_file(settings.SETTINGS["paths"]["BRINGME_PATH"])
+if not CONFIG:
+    CONFIG = utilities.load_toml_file(settings.SETTINGS["paths"]["BRINGME_DEFAULTS_PATH"])
+if not CONFIG:
+    # logger.warn("Could not load bringme defaults")
+    print("Could not load bringme defaults")
+    
+
+def explorer_bring_it(folder_path):
+    Key("c-l/5").execute()
+    Text("{}".format(folder_path)).execute()
+    # Alternate method: is faster but somewhat inconsistent.
+        # if not paste_string_without_altering_clipboard(folder_path):
+        #     print("failed to paste {}".format(folder_path))
+        # Pause("10").execute()
+    # Second alternate method: is both fast and consistent, but alters the clipboard
+        # pyperclip.copy(folder_path)
+        # Pause("5").execute()
```

Comment:
I have found that the current method is quite fast. Let's avoid dependencies where practical. Let's remove this content that is commented out.